### PR TITLE
Movie details layout 

### PR DESCRIPTION
**Pull request by AlekseyYakovlev**
@AlekseyYakovlev
1. Build scripts are changed to .kts
2. Added Movie details layout 
3. Added Custom RatingBar with vector drawable support

### CODE REVIEW

File: app/src/main/res/drawable/ic_baseline_arrow_back_ios_24.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="10dp"
+    android:height="10dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:tint="?attr/colorControlNormal">
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M11.67,3.87L9.9,2.1 0,12l9.9,9.9 1.77,-1.77L3.54,12z"/>
```
Comment (DmitryKolpo) [inline app/src/main/res/drawable/ic_baseline_arrow_back_ios_24.xml]: Иконка отличается от дизайна.

Из фигмы можно экспортировать svg иконку и заимпортить в студию new>vector assets.

File: app/src/main/res/layout/movie_details.xml [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+            android:text="@string/back"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            app:drawableStartCompat="@drawable/ic_baseline_arrow_back_ios_24"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+
+        <TextView
+            android:id="@+id/tv_pg"
```
Comment (DmitryKolpo) [inline app/src/main/res/layout/movie_details.xml:55]: Текст по дизайну внутри квадратной вьюшки с тенью. И там отступ немного другой получается от имени фильма.

File: app/src/main/res/layout/movie_details.xml [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+            app:layout_constraintTop_toBottomOf="@+id/iv_poster" />
+
+        <TextView
+            android:id="@+id/tv_genre"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/space_16"
+            android:layout_marginTop="4dp"
+            android:text="Action, Adventure, Fantasy"
+            android:textColor="#FF3466"
```
Comment (DmitryKolpo) [inline app/src/main/res/layout/movie_details.xml]: Лучше вынести в colors

File: app/src/main/res/values-night/themes.xml [hunk 4]
```diff
@@ -1,6 +1,12 @@
-<resources xmlns:tools="http://schemas.android.com/tools">
+<resources>
     <!-- Base application theme. -->
-    <style name="Theme.AndroidAcademy2020" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+    <style name="Theme.AndroidAcademy2020" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+<!--        <item name="windowNoTitle">true</item>-->
+<!--        <item name="windowActionBar">false</item>-->
+
+<!--        <item name="android:windowTranslucentStatus">true</item>-->
+<!--        <item name="android:windowTranslucentNavigation">true</item>-->
```
Comment (DmitryKolpo) [inline app/src/main/res/values-night/themes.xml]: Cleanup code

Стоит удалять неиспользуемый код.

File: app/src/main/res/values-night/themes.xml [hunk 5]
```diff
@@ -10,7 +16,8 @@
         <item name="colorSecondaryVariant">@color/teal_200</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
+<!--        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>-->
```
Comment (DmitryKolpo) [inline app/src/main/res/values-night/themes.xml]: Cleanup code

File: app/src/main/res/values/themes.xml [hunk 6]
```diff
@@ -1,6 +1,12 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.AndroidAcademy2020" parent="Theme.MaterialComponents.DayNight.DarkActionBar">
+    <style name="Theme.AndroidAcademy2020" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+        <!--        <item name="windowNoTitle">true</item>-->
+        <!--        <item name="windowActionBar">false</item>-->
+
+        <!--        <item name="android:windowTranslucentStatus">true</item>-->
+        <!--        <item name="android:windowTranslucentNavigation">true</item>-->
```
Comment (DmitryKolpo) [inline app/src/main/res/values/themes.xml]: Cleanup code

File: app/src/main/res/values/themes.xml [hunk 7]
```diff
@@ -10,7 +16,58 @@
         <item name="colorSecondaryVariant">@color/teal_700</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
+        <!--        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>-->
```
Comment (DmitryKolpo) [inline app/src/main/res/values/themes.xml]: Cleanup code

File: app/src/main/res/drawable/ic_baseline_arrow_back_ios_24.xml (hunk 1 above)
Comment (AlekseyYakovlev) [reply app/src/main/res/drawable/ic_baseline_arrow_back_ios_24.xml]: Done commit [https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/18fdcea8c33d89869aca6232ce43aae0276ab205](https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/18fdcea8c33d89869aca6232ce43aae0276ab205)

File: app/src/main/res/layout/movie_details.xml (hunk 2 above)
Comment (AlekseyYakovlev) [reply app/src/main/res/layout/movie_details.xml:55]: Добавил elevation, поменял отступ. Commit [https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/b121183cc608f79270456c28a4ca32ad4e440940](https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/b121183cc608f79270456c28a4ca32ad4e440940)

File: app/src/main/res/layout/movie_details.xml (hunk 3 above)
Comment (AlekseyYakovlev) [reply app/src/main/res/layout/movie_details.xml]: Done. Commit [https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9](https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9)

File: app/src/main/res/values-night/themes.xml (hunk 4 above)
Comment (AlekseyYakovlev) [reply app/src/main/res/values-night/themes.xml]: Done. Commit [https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9](https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9)

File: app/src/main/res/values-night/themes.xml (hunk 5 above)
Comment (AlekseyYakovlev) [reply app/src/main/res/values-night/themes.xml]: Done. Commit [https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9](https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9)

File: app/src/main/res/values/themes.xml (hunk 6 above)
Comment (AlekseyYakovlev) [reply app/src/main/res/values/themes.xml]: Done. Commit [https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9](https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9)

File: app/src/main/res/values/themes.xml (hunk 7 above)
Comment (AlekseyYakovlev) [reply app/src/main/res/values/themes.xml]: Done. Commit [https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9](https://github.com/AlekseyYakovlev/AndroidAcademy2020/pull/2/commits/e976abfe5a884638cb19d17b8c9e6646ff31a4d9)

**Review by DmitryKolpo — APPROVED** [review]

**Pull request merged**